doc/backend_vars.asciidoc

```diff
@@ -289,3 +289,15 @@ AMT_PASSWORD;string;;Password for admin AMT user on target host
 Variable;Values allowed;Default value;Explanation
 WORKER_HOSTNAME;string;undef;Worker host name
 |====================
+
+.SPVM backend
+[grid="rows",format="csv"]
+[options="header",cols="^m,^m,^m,v",separator=";"]
+|====================
+Variable;Values allowed;Default value;Explanation
+WORKER_HOSTNAME;string;undef;Worker host name
+NOVALINK_HOSTNAME;string;undef;Novalink target host to connect to
+NOVALINK_USERNAME;string;undef;Username to authenticate on Novalink host
```

Comment:
Isn't there a default 'root' according to https://github.com/os-autoinst/os-autoinst/pull/1790/files#diff-b3e78e478b3e4f99f0291d2d0f49ea3d73babcf5c712d50f9d6ea770af388bf3R32?

```suggestion
NOVALINK_USERNAME;string;root;Username to authenticate on Novalink host
```

Reply:
fixed as part of https://github.com/os-autoinst/os-autoinst/pull/1845